src/Discord.Net.Rest/Entities/Messages/RestMessage.cs

```diff
@@ -80,6 +80,12 @@ public abstract class RestMessage : RestEntity<ulong>, IMessage, IUpdateable
         /// <inheritdoc/>
         public MessageType Type { get; private set; }
 
+
+        /// <summary>
+        ///     Gets the thread that was started from this message.
+        /// </summary>
+        public RestThreadChannel Thread { get; private set; }
```

Comment:
Null if it doesn't exist? Add a remark defining that behavior

Reply:
I've pushed a fix... should be resolved? 
*I was really struggling to write a good enough comment and i'm really not sure how to describe it in a better way*